src/store/TranscriptionsStore.js

```diff
@@ -439,9 +439,9 @@ const TranscriptionsStore = types.model('TranscriptionsStore', {
     return count;
   },
 
-  get lockedByCurrentUser () {
+  get lockedByDifferentUser () {
     const user = getRoot(self).auth.userName
```

Comment:
This is very odd.

For my staging user account, my details are...

```
{
  login: 'darkeshard',
  display_name: 'Darke Shard'
}
```

The `ToveTranscriptionResource.lockedby` value keeps track of the user's **login,** but `auth.userName` is pulling the user's **display name.**

Question: does the account you use for testing have login === display_name?

Reply:
Welp. You might've found the issue. Let me spin this up and take a look.

Reply:
I think that's the fix we need. Why do we have `login` and `display_name`?